PoseEstimation-TFLiteSwift/PoseConfidenceMapDrawingView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  PoseConfidenceMapDrawingView.swift
+//  PoseEstimation-TFLiteSwift
+//
+//  Created by Doyoung Gwak on 2020/04/24.
+//  Copyright © 2020 Doyoung Gwak. All rights reserved.
+//
+
+import UIKit
+
+class PoseConfidenceMapDrawingView: UIView {
+    typealias CGLine = (from: CGPoint, to: CGPoint)
+    var outputChannelIndexes: [Int] = [0]
+    var output: TFLiteFlatArray<Float32>? {
+        didSet {
+            self.setNeedsDisplay()
+        }
+    }
+
+    override func draw(_ rect: CGRect) {
+        guard let output = output else { return }
+        
+        let (rowCount, colCount) = (output.dimensions[1], output.dimensions[2])
+        let (width, height) = (frame.width, frame.height)
+        let (oneAreaWidth, oneAreaHeight) = (width/CGFloat(colCount), height/CGFloat(rowCount))
+        
+        // top-left is (0,0)
+        for row in 0..<rowCount {
+            for col in 0..<colCount {
+                var value = outputChannelIndexes.reduce(0.0) { value, outputChannelIndex in
+                    return value + output[0, row, col, outputChannelIndex]
+                }
+                value = min(max(value, -1.0), 1.0)
+                let drawingAreaRect = CGRect(x: oneAreaWidth*CGFloat(col), y: oneAreaHeight*CGFloat(row),
+                                             width: oneAreaWidth, height: oneAreaHeight)
+                drawRect(with: drawingAreaRect, value: value)
+            }
+        }
+    }
+    
+    func drawRect(with rect: CGRect, value: Float32 = 0.0) {
```

Comment:
'value' is so abstract name.

I recommend to pass UIColor to drawRect function instead (value: Float32).
drawRect function have to no idea about DrawingConstant.

Reply:
It could be changed to `alpha`, `confidence`, or `score`. 
Thanks for your opinion.

Reply:
@syjdev 
I changed to `componentOfVector` 🙂